Timeout when download takes too long

The CVRPLIB website has been offline the entire day. The code currently just hangs waiting for a response. I think we should throw an error if downloading hasn't finished after 20 seconds.

Using the `timeout` argument:
```python
    response = urlopen(url, timeout=60)
```
we get this error when it times out:
```py
E               urllib.error.URLError: <urlopen error timed out>
```
Should we catch the `URLError` and describe something more informative?

--- a/vrplib/download/download_instance.py
+++ b/vrplib/download/download_instance.py
@@ -15,9 +15,13 @@ def download_instance(name: str, path: str):
         listed in `vrplib.list_instances()`.
     path
         The path where the instance file should be saved.
+
+    Raises
+    ------
     """
     ext = "txt" if is_vrptw(name) else "vrp"
-    response = urlopen(CVRPLIB_URL + f"{find_set(name)}/{name}.{ext}")
+    url = CVRPLIB_URL + f"{find_set(name)}/{name}.{ext}"
+    response = urlopen(url, timeout=30)
 
     instance_text = response.read().decode("utf-8")
 

--- a/vrplib/download/download_instance.py
+++ b/vrplib/download/download_instance.py
@@ -15,9 +15,6 @@ def download_instance(name: str, path: str):
         listed in `vrplib.list_instances()`.
     path
         The path where the instance file should be saved.
-
-    Raises
-    ------
     """
     ext = "txt" if is_vrptw(name) else "vrp"
     url = CVRPLIB_URL + f"{find_set(name)}/{name}.{ext}"

--- a/vrplib/download/download_solution.py
+++ b/vrplib/download/download_solution.py
@@ -16,7 +16,9 @@ def download_solution(name: str, path: str):
     path
         The path where the solution file should be saved.
     """
-    response = urlopen(CVRPLIB_URL + f"{find_set(name)}/{name}.sol")
+    url = CVRPLIB_URL + f"{find_set(name)}/{name}.sol"
+    response = urlopen(url, timeout=30)
+
     solution_text = response.read().decode("utf-8")
 
     with open(path, "w") as fi:

